fastjet/internal/delaunay/delaunay.go

```diff
@@ -69,6 +69,7 @@ func (d *Delaunay) Triangles() []*Triangle {
 // whose nearest neighbor changed due to the insertion. The slice may contain
 // duplicates.
 func (d *Delaunay) Insert(p *Point) (updatedNearestNeighbor []*Point) {
+	p.adjacentTriangles = make(triangles, 0)
```

Comment:
instead of (implicitly) re-allocating, perhaps first test whether `adjacentTriangles` is of `len` 0 (and then call `make`) otherwise:
```go
p.adjacentTriangles = p.adjacentTriangles[:0]
```
so we re-use the backing storage of the slice ?